io-scenario-quality-checker.iml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you remove ioprojectarchitecture.iml from repo and add *.iml rule to .gitignore? I forgot to add that rule and now we get polluted by intellij stuff :)

Reply:
I think it's done now :)